tests/benchmark/results.md

```diff
@@ -106,3 +106,21 @@ The benchmark was run as a Kubernetes job in GKE:
 | postgres_conn_benchmark | one_gb     | 5.56min      | 62.5MB       | 14.07s          | 1.14s             |
 | postgres_conn_benchmark | five_gb    | 24.44min     | 78.15MB      | 1.34min         | 5.73s             |
 | postgres_conn_benchmark | ten_gb     | 45.64min     | 61.71MB      | 2.37min         | 11.48s            |
+
+### Database S3 to Bigquery using default path
+
+|  database | dataset    | total_time   | memory_rss   | cpu_time_user   | cpu_time_system   |
+|:----------|:-----------|:-------------|:-------------|:----------------|:------------------|
+| bigquery  | hundred_kb | 18.64s       | 54.51MB      | 20.45ms         | 5.73ms            |
+| bigquery  | one_gb     | 1.18hrs      | 6.56GB       | 8.4s            | 1.59s             |
+| bigquery  | ten_kb     | 19.02s       | 53.84MB      | 17.9ms          | 5.2ms             |
+| bigquery  | ten_mb     | 47.02s       | 201.8MB      | 32.79ms         | 11.09ms           |
+
+### Database S3 to Bigquery using native path
+
+| database   | dataset    | total_time   | memory_rss   | cpu_time_user   | cpu_time_system   |
+|:-----------|:-----------|:-------------|:-------------|:----------------|:------------------|
+| bigquery   | hundred_kb | 2.92min      | 56.57MB      | 41.94ms         | 35.83ms           |
+| bigquery   | one_gb     | 4.25min      | 66.32MB      | 54.88ms         | 48.01ms           |
+| bigquery   | ten_kb     | 2.9min       | 55.51MB      | 40.74ms         | 34.94ms           |
+| bigquery   | ten_mb     | 4.17min      | 57.6MB       | 46.64ms         | 46.83ms           |
```

Comment:
@utkarsharma2, please, if possible, document why we didn't use the standard benchmarking script on K8s in this document.

Also, please,  log a ticket so we can further investigate the NDJSON issues you faced while running it.

Reply:
@utkarsharma2 please, document in the `results.md file` the exact problems you had with which datasets and what changes someone would need to be able to make to reproduce these results.

Reply:
If possible, please, share here the link to the Github issue related to these problems - so we can address them after 1.0.0 is released.